src/pages/rdfrank.html

```diff
@@ -1,7 +1,7 @@
 <link href="css/rdfrank.css" rel="stylesheet">
 <link href="js/lib/ng-tags-input/ng-tags-input.min.css" rel="stylesheet"/>
 
-<div class="page">
+<div id="rdf_rank" class="page">
```

Comment:
uhmm why the underscore?

Reply:
No reason really. I usually write IDs with underscore while classes with dashes

Reply:
Will change it to a dash

Reply:
Usually classess are with dash and ids are camelcase

Reply:
Changed to camelcase